faker.js

```diff
@@ -46,7 +46,7 @@ const randomName = () => {
   return submitName.trim();
 };
 // set amount of random districts
-const districts = [faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}'), faker.fake('{{lorem.word}}'), faker.fake('{{lorem.word}}'), faker.fake('{{lorem.word}}')];
+const districts = [faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}'), faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}'), faker.fake('{{lorem.word}}')];
```

Comment:
*Just a suggestion*
If you are trying to just generate a district with one word and another district with more than one word, 
faker.random.arrayElement([faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}')]) 
is the same with 
[faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}'), faker.fake('{{lorem.word}}'), faker.fake('{{lorem.words}}'), faker.fake('{{lorem.word}}')].